.changeset/cold-lions-turn.md

```diff
@@ -0,0 +1,40 @@
+---
+'@openfn/language-progres': major
+---
+
+- Migrate from `common.http` to `common.request`.
+- Remove `axios` and `nock` from progres.
+- Remove exportation of `http` from `common`, and the exportation of `axios`.
+- With the switch to `common-request` the function pattern changes from:
```

Comment:
Does it? What's this change all about?

Ah - I think you've fixed a docs bug but that's very. We don't need to mention that here. The old docs are wrong.

You can remove this part of the changelog

Reply:
I have removed it